engine/src/main/java/org/terasology/monitoring/impl/PerformanceMonitorImpl.java

```diff
@@ -40,8 +41,10 @@
 public class PerformanceMonitorImpl implements PerformanceMonitorInternal {
     private static final int RETAINED_CYCLES = 60;
     private static final double DECAY_RATE = 0.98;
+    private static final Activity OFF_THREAD_ACTIVITY = new NullActivity();
```

Comment:
Got it. Should I then submit a PR with the javadoc class for NullActivity:

"NullActivity instances are used for processes NOT running on the main thread. Not strictly necessary (these processes are ignored by the PerformanceMonitor anyway) an instance of this class offers a slight performance improvement over standard Activity implementations as it doesn't call the PerformanceMonitor.endActivity() method."